articles/active-directory-b2c/user-flow-custom-attributes.md

```diff
@@ -143,9 +143,12 @@ You can create these attributes by using the portal UI before or after you use t
 
 |Name     |Used in |
 |---------|---------|
-|`extension_loyaltyId`  | Custom policy|
+|`extension_loyaltyId` | Custom policy|
 |`extension_<b2c-extensions-app-guid>_loyaltyId`  | [Microsoft Graph API](microsoft-graph-operations.md#application-extension-directory-extension-properties)|
 
+> [!NOTE] 
+> When using custom attributes in custom policies, the claim type ID must be prefixed with `extension_` to allow the correct data mapping to take place within the Azure AD B2C directory.
```

Comment:
```suggestion
> When using a custom attribute in custom policies, you must prefix the claim type ID with `extension_` to allow the correct data mapping to take place within the Azure AD B2C directory.
```

Reply:
@bolt-io 
Thank you for your contribution. 
See my proposed correction. Please commit the suggestion so that we sign off your PR